src/Sulu/Bundle/SecurityBundle/Security/Exception/NoTokenFoundException.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\Security\Exception;
+use Sulu\Bundle\SecurityBundle\Entity\User;
+
+/**
+ * This exception is thrown when a token-email for user without a token is requested
+ * @package Sulu\Bundle\SecurityBundle\Security\Exception
+ */
+class NoTokenFoundException extends SecurityException
```

Comment:
This exception is not thrown if at some point the program requests a token an it doesn't exist. This exception is thrown if an email is requested for a user who doesn't have a token. So its not like that a well specified token is not found but that just no token was found for that user. In my opinion TokenNotFoundException is a misleading name in this case, because it says that a well specified token was not found